docs/tutorials/complete_applications/complete_application/step_04/step_04.md

```diff
@@ -0,0 +1,181 @@
+and predictions generated using both machine learning and baseline methods.
+
+![Scenario Page](result.png){ width=700 style="margin:auto;display:block;border: 4px solid rgb(210,210,210);border-radius:7px" }"
+
+The Scenario Page is constructed using a combination of Markdown and Python code. Below is a detailed explanation of each component:
+
+# Markdown (pages/scenario/scenario.md)
+
+```markdown
+# Create your scenario:
+
+<|layout|columns=3 1 1 1 1|
+<|{scenario}|scenario_selector|>
+
+**Prediction date** <br/>
+<|{day}|date|active={scenario}|not with_time|>
+
+**Max capacity** <br/>
+<|{max_capacity}|number|active={scenario}|>
+
+**Number of predictions** <br/>
+<|{n_predictions}|number|active={scenario}|>
+
+<br/> <|Save|button|on_action=save|active={scenario}|>
+|>
+ 
+<|{scenario}|scenario|on_submission_change=submission_change|>
+
+<|{predictions_dataset}|chart|x=Date|y[1]=Historical values|type[1]=bar|y[2]=Predicted values ML|y[3]=Predicted values Baseline|>
+
+# Data Node Exploration
+
+<|layout|columns=1 5|
+<|{data_node}|data_node_selector|>
+
+<|{data_node}|data_node|>
+|>
+```
+
+The Markdown section outlines the arrangement and elements of the Scenario Page. It consists of the following components:
+
+- **Scenario Selector**: `<|{scenario}|scenario_selector|>`
+
+A component (a dropdown) that allows users to select different scenarios. The selected scenario will affect the values of other components on the page.
+
+- **Prediction Date**: `<|{day}|date|...|>`
+
+A date picker where users can select the date for which they want to make predictions. The selected date will be used for both machine learning and baseline predictions.
+
+- **Max Capacity**: `<|{max_capacity}|number|...|>`
+
+A number input field where users can set the maximum capacity value. This value is used to cap the predictions if they exceed the specified maximum.
+
+- **Number of Predictions**: `<|{n_predictions}|number|...|>`
+
+A number input field where users can set the desired number of predictions to be made.
+
+- **Save Button**: `<|Save|button|on_action=save|active={scenario}|>`
+
+A button that triggers the "save" action when clicked. It is used to save the selected scenario and parameter values.
+
+- **Scenario Section**: `<|{scenario}|scenario|on_submission_change=submission_change|>`
+
+A section that displays information about the currently selected scenario. It includes details about the scenario, properties, and the ability to delete or submit the scenario. When the scenario status is changed, `submission_change` is called with information on the scenario.
+
+- **Predictions Chart**: `<|{predictions_dataset}|chart|...|>`
+
+A chart that displays historical values and the predicted values obtained from machine learning and baseline methods. It shows how well the predictions align with the historical data.
+
+- **Data Node Exploration**: `<|{data_node}|data_node_selector|> <|{data_node}|data_node_selector|>`
```

Comment:
```suggestion
- **Data Node Exploration**: `<|{data_node}|data_node_selector|>`
```

Reply:
It should be data_node for the second one

Reply:
AH yes, indeed.